Override electron to 40.6.0.

@generated
@generator: scripts/apply-override.sh --patch common/override-electron.diff --override 'direct-dev:electron=40.6.0'

Index: code-editor-src/package.json
===================================================================
--- code-editor-src.orig/package.json
+++ code-editor-src/package.json
@@ -173,7 +173,7 @@
     "css-loader": "^6.9.1",
     "debounce": "^1.0.0",
     "deemon": "^1.13.6",
-    "electron": "39.6.0",
+    "electron": "40.6.0",
     "eslint": "^9.36.0",
     "eslint-formatter-compact": "^8.40.0",
     "eslint-plugin-header": "3.1.1",
